File: Group 85-b/level 35/homework/hw2.py
#3) ფუნქცია იღებს სიას რიცხვებისგან და აბრუნებს მხოლოდ დადებითების ჯამს.
def sum_of_positives(numbers):
    positive_sum = 0
    for num in numbers:
        if num > 0:
            positive_sum += num
    return positive_sum

File: Group 85-b/level 55/homework/hw2.js

//დაწერეთ კოდი, რომელიც აიღებს მასივს [10,15,20,25], თუ რიცხვი მეტია 18-ზე, გამოიტანოს "yes", სხვა შემთხვევაში "no"

let numbers = [10,15,20,25];
for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] > 18) {
        console.log("yes");
    } else {
        console.log("no");
    }
}
